shared/src/test/scala/amf/core/render/GovernanceModeTest.scala

```diff
@@ -69,4 +70,30 @@ class GovernanceModeTest extends AsyncFunSuite with FileAssertionTest {
       result <- assertDifferences(file, golden)
     } yield result
   }
+
+  test("lists should be rendered with @list when using Governance Mode with Flattened JsonLd") {
```

Comment:
If this is a test for `@list` then it is weird that the golden file has the old notation. If this test is for the next commit then it should be in the next commit.

Reply:
done